Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -2161,8 +2161,13 @@ bool addNew
             var path = factory.Configuration.PrefixPath;
             if (removeFromStorage && Directory.Exists(path)) {
                 var t = Task.Run(() => {
-                    Directory.Delete(path, true);
-                    return true;
+                    try {
+                        Directory.Delete(path, true);

```

Comment:
I'm confused - why wouldn't we want to display a message box here if delete fails? Shouldn't the user know?